src/main/charts/bamboo/values.yaml

```diff
@@ -340,7 +340,105 @@ ingress:
 # Bamboo configuration 
 #
 bamboo:
-  
+
+  # -- Supply the base URL for the installation.
+  #
+  baseUrl:
+
+  # -- Override the server/agent broker URL; this is optional.
+  #
+  brokerUrl:
+
+  # The security token with which the agent will authenticate to the
+  # Bamboo server. The token should be set to a 40-character
```

Comment:
@achystoprudov Can you confirm this is the correct format?

Reply:
@tarka , at Bamboo JavaDocs we wrote
```
*  Sets token value for Agent authentication (if defined bamboo won't generate new token).
 *  Token should contain only hexadecimal characters and have 40 characters length.
*  <li>Usage: <ul><li>env : bamboo.setup.remote.agent.security.token.value="ABCDEF12345678901234abcdef0123456789abcd"
*                  <li>java: -Dbamboo.setup.remote.agent.security.token.value=ABCDEF12345678901234abcdef0123456789abcd
*             </ul>
*  </ul>
```